packages/calypso-analytics/src/index.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Re-export
+ */
+export { getDoNotTrack } from './utils/do-not-track';
```

Comment:
Looks like `getDoNotTrack` is the default export: https://github.com/Automattic/wp-calypso/pull/39001/files#diff-6dd963982aebbb28e8af6ba227530323R11

Reply:
```
export { default as getDoNotTrack } from './utils/do-not-track';
```

Seems to work for me

Reply:
thanks!